yarn-project/noir-protocol-circuits-types/src/index.ts

```diff
@@ -148,268 +146,171 @@ export async function executeTail(
 }
 
 /**
- * Executes the public kernel in the setup phase.
- * @param publicKernelPrivateInputs - The public kernel setup circuit private inputs.
- * @returns The public inputs.
+ * Converts the inputs to the base rollup circuit into a witness map.
+ * @param inputs - The base rollup inputs.
+ * @returns The witness map
  */
-export async function executePublicKernelSetup(
-  publicKernelPrivateInputs: PublicKernelCircuitPrivateInputs,
-): Promise<PublicKernelCircuitPublicInputs> {
-  const params: PublicSetupInputType = {
-    input: mapPublicKernelCircuitPrivateInputsToNoir(publicKernelPrivateInputs),
-  };
-
-  const returnType = await executePublicKernelSetupWithACVM(params);
-
-  return mapPublicKernelCircuitPublicInputsFromNoir(returnType);
+export function convertBaseRollupInputs(inputs: BaseRollupInputs): WitnessMap {
```

Comment:
Nit: I'd keep the `FromNoir`/`ToNoir` suffixes to methods here for clarity.

Reply:
Used From/To WitnessMap.